demo/launch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+CWD=`pwd`
+CONFIG=${CWD}/happi_demo.cfg
+TOOLBAR=${CWD}/demo_toolbar.yml
```

Comment:
Found this while trying to run the example:
```suggestion
TOOLBAR=${CWD}/example_toolbar.yml
```

Reply:
Good catch...